week-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,54 @@
-function setAlarm() {}
+let intervalId;
+let bgIntervalId;
+
+function setAlarm() {
+  clearInterval(intervalId);
+  clearInterval(bgIntervalId);
+  setBackground("none");
+  document.getElementById("alarmSet").classList.remove("invalidInput");
+
+  let alarmSet = document.getElementById("alarmSet").value;
+
+  if (alarmSet > 3599) {
+    document.getElementById("alarmSet").classList.add("invalidInput");
```

Comment:
This is a nice touch - giving the user clear signals when they've done something wrong is really important